src/src/CustomBuild/ESPEasy_buildinfo.h

```diff
@@ -20,7 +20,7 @@
 #endif // if defined(ESP32)
 
 
-#define BUILD                           20112    // git version e.g. "20103" can be read as "2.1.03" (stored in int16_t)
+#define BUILD                           20113    // git version e.g. "20103" can be read as "2.1.03" (stored in int16_t)
```

Comment:
@TD-er should I bump the build again, or will it 'fit' into the current (new) number?

Reply:
You now rely on the given default which is guaranteed to "0" for that bit.
So then you don't need to increment it. The current 'mega' branch has already this build nr.

Reply:
> The current 'mega' branch has already this build nr.

That's why I'm asking, I bumped it initially, but you made the same increment.
- I'm not changing the settings struct (only using a bit in an existing member)
- Don't need a conversion

Conclusion -> No need to increment 👍